src/test/java/io/airlift/bytecode/expression/BytecodeExpressionAssertions.java

```diff
@@ -38,7 +39,7 @@
 
 public final class BytecodeExpressionAssertions
 {
-    private static final boolean DUMP_BYTECODE_TREE = false;
+    public static final AtomicBoolean DUMP_BYTECODE_TREE = new AtomicBoolean();
```

Comment:
The commit message says "Always dump tree in BytecodeExpressionAssertions", but the value is false by default.

Reply:
Thanks, not sure what I was thinking when I wrote that. Will update.